source/appModules/lockapp.py

```diff
@@ -90,10 +90,10 @@ def event_appModule_gainFocus(self):
 			# Move the review cursor so others can't access its previous position.
 			self._oldReviewPos = api.getReviewPosition()
 			self._oldReviewObj = self._oldReviewPos.obj
-			api.setNavigatorObject(eventHandler.lastQueuedFocusObject)
+			api.setNavigatorObject(eventHandler.lastQueuedFocusObject, isFocus=True)
 
 	def event_appModule_loseFocus(self):
 		if not config.conf["reviewCursor"]["followFocus"]:
-			api.setReviewPosition(self._oldReviewPos)
+			api.setReviewPosition(self._oldReviewPos, isCaret=False)

```

Comment:
As isCaret is False by default,   explicitly  specifying isCaret=False here is not needed, and is perhaps a little confusing. I'd prefer that anywhere in the codebase that  isCaret is only ever specified if it needs to be set to true. Mirroring that of isFocus for setNavigatorObject.